docs/pages/includes/install-linux.mdx

```diff
@@ -41,52 +38,50 @@ Use the appropriate commands for your environment to install your package.
   ```
 
   </TabItem>
-  <TabItem label="Amazon Linux 2/RHEL (RPM)" scope="oss">
+  <TabItem label="AL 2/RHEL 7 (yum)" scope="oss">
```

Comment:
would keep as Amazon Linux 2 to avoid confusion since some may not be aware of what AL means.